OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -225,16 +225,10 @@ public override Activity Tick(Actor self)
 			{
 				aircraft.MakeReservation(dest);
 
-				if (aircraft.Info.VTOL)
-				{
-					if (aircraft.Info.TurnToDock)
-						QueueChild(self, new Turn(self, aircraft.Info.InitialFacing), true);
-
-					QueueChild(self, new HeliLand(self, true, dest), true);
-				}
-				else
-					QueueChild(self, new Land(self, Target.FromPos(dest.CenterPosition + offset), true, dest), true);
+				if (aircraft.Info.VTOL && aircraft.Info.TurnToDock)
+					QueueChild(self, new Turn(self, aircraft.Info.InitialFacing), true);
```

Comment:
These `Turn`s are logically part of the landing behaviour, so really belong inside the `Land` activity.

This is already done in #16509, so we don't need to address this here.